documentation/book/assembly-customizing-deployments.adoc

```diff
@@ -22,8 +22,10 @@ include::con-customizing-labels-and-annotations.adoc[leveloffset=+1]
 
 include::con-customizing-pods.adoc[leveloffset=+1]
 
+include::con-customizing-containers.adoc[leveloffset=+1]
+
 include::con-customizing-image-pull-policy.adoc[leveloffset=+1]
 
 include::con-customizing-pod-disruption-budgets.adoc[leveloffset=+1]
 
-include::proc-customizing-deployments.adoc[leveloffset=+1]
\ No newline at end of file
+include::proc-customizing-deployments.adoc[leveloffset=+1]
```

Comment:
what's changed here?

Reply:
That is a good question?

Reply:
End of line / new line at the end of the file perhaps? It is a whitespace change.